src/screens/TopicPageScreen/index.js

```diff
@@ -20,7 +20,6 @@ import useViewPostTimeHook from '../FeedScreen/hooks/useViewPostTimeHook';
 import useOnBottomNavigationTabPressHook, {
   LIST_VIEW_TYPE
 } from '../../hooks/navigation/useOnBottomNavigationTabPressHook';
-import {COLORS} from '../../utils/theme';
 import {Context} from '../../context';
 import {downVote, upVote} from '../../service/vote';
 import {getFeedDetail} from '../../service/post';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `COLORS` from the import statement suggests that it's no longer being used in this file. This is a good practice as it helps to reduce unnecessary imports and keeps the code clean.
```diff
- import {COLORS} from '../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->